source/PhysicalQuantity/Temperature.php

```diff
@@ -53,7 +55,9 @@ function ($x) {
             }
         );
         $newUnit->addAlias('F');
+        $newUnit->addAlias('f');
```

Comment:
that'd make sense, I'll adjust the code to be case insensitive.  I had added that because I noticed it failed if "f" was sent in, it wasn't being case insensitive like it should be.